P5/Source/Specs/schemaSpec.xml

```diff
@@ -31,20 +35,40 @@ $Id$
   </classes>
   <content>
     <sequence>
-      
-        <alternate minOccurs="0" maxOccurs="unbounded">
-          <classRef key="model.glossLike"/>
-          <classRef key="model.descLike"/>
-        </alternate>
-      
-      
-        <alternate minOccurs="0" maxOccurs="unbounded">
-          <classRef key="model.oddRef"/>
-          <classRef key="model.oddDecl"/>
-        </alternate>
-      
+      <alternate minOccurs="0" maxOccurs="unbounded">
+        <classRef key="model.glossLike"/>
+        <classRef key="model.descLike"/>
+      </alternate>
+      <alternate minOccurs="0" maxOccurs="unbounded">
+        <classRef key="model.oddRef"/>
+        <classRef key="model.oddDecl"/>
+      </alternate>      
     </sequence>
   </content>
+  <constraintSpec scheme="schematron" ident="deprecate_schemaSpec_in_bizarre_places">
+    <desc>enforces the depecration of <gi>schemaSpec</gi> as a child
+    of anything other than <gi>front</gi>, <gi>body</gi>,
```

Comment:
please add <encodingDesc> here as well.

Reply:
what I wanted to say here is please add `<encodingDesc>`

Reply:
Good catch! Thank you. Fixing now …